nav2_msgs/action/FollowWaypoints.action

```diff
@@ -1,4 +1,6 @@
 #goal definition
+uint32 number_of_loops
```

Comment:
Initially I handled them in the panel level. I just appended the waypoints depending on the number of loops. So say for example you have three waypoints and 3 loops, we totally have 9 waypoints. When you start the navigation and the robot reaches the 1st waypoint again, then in the terminal you see the logs, stating that you are reaching 4th waypoint, which totally did not make sense, because it typically is the 1st waypoint and might cause some sort of confusion if you have large number of waypoints.

Reply:
> stating that the robot is reaching 4th waypoint

I think that's fine in context, but let me take a look at this again with a fresh mind that this is going to involve some changes in the WP follower. It just wasn't my initial expectation